app/tasks/doctor.php

```diff
@@ -135,8 +139,15 @@
 
         try {
             $mail = $register->get('smtp'); /* @var $mail \PHPMailer\PHPMailer\PHPMailer */
+            if (empty($receiver)) {
+                $mail->addAddress($receiver);
+            } elseif (strtolower($interactive === "y")) {
+                $receiver = Console::confirm('Enter your email address to test SMTP connection: ');
+                $mail->addAddress($receiver);
+            } else {
+                $mail->addAddress('demo@example.com', 'Example.com');
+            }
```

Comment:
If the interactive option is set to 'Y' and a value for `$receiver` is also provided, what will occur?